Training/box_it.cpp

```diff
@@ -1,100 +1,108 @@
 #include <iostream>
 #include "box_it.h"
-
 using namespace std;
 
 class Box {
 private:
-    int m_l = 0,  // length
-        m_b = 0,  // breadth
-        m_h = 0;  // height
+	int m_l = 0;  // length
+	int m_b = 0;  // breadth
+	int m_h = 0;  // height
+
 public:
-    Box() {}
-    Box(int length, int breadth, int height)
-    {
-        m_l = length;
-        m_b = breadth;
-        m_h = height;
-    }
-    Box(const Box &b)
-    {
-        m_l = b.m_l;
-        m_b = b.m_b;
-        m_h = b.m_h;
-    }
-    int getLength() { return m_l; }
-    int getBreadth() { return m_b; }
-    int getHeight() { return m_h; }
-    long long CalculateVolume() { return ((long long)m_l * m_b) * m_h; }
+	Box() {}
+
+	/// Es-Note: Initializer lists call the constructors of the data members,
+	/// this allows for instantiation to occur on members, instead of construction
+	/// followed by assignment.
+	Box (int length, int breadth, int height) : m_l(length), m_b(breadth), m_h(height) {
+	}
+
+	/// Es-Note: You can re-use constructors like this:
+	Box (const Box & b) : Box(b.m_l, b.m_b, b.m_h) {
+	}
+
+	int getLength ()  const  { return m_l; }
```

Comment:
Is this a new format guideline? You didn't add a space between function name and parameters before.

Reply:
Oh-- Hmm... I guess I didn't do it on the last PR because I didn't want to be invasive?

The style isn't super important. The space makes it easier to search for definition/declaration vs. usage (since you can't have a space there).

Reply:
That's interesting. I'll revise the existing code to match this style.